### DISCUSSION TITLE
Updated Lychee 5.2 exporting Mars 4 Max files that UVtools cannot open

### DISCUSSION THREAD

**Discussion by bryanseger** [sn4k3/UVtools#710] [Q&A]
I have updated to the newest version of Lychee that supports both Mars 4 models. Printing a Mars 4 Max file directly from Lychee seems to run fine. But when I try to open that file in UVtools, I get the below error. Tango and UVtools seems to play nicely with Mars 4 Max, but as of right now I can't get UVtools and Lychee to cooperate. I posted this on Lychee's Discord too....not sure if the issue is on their end or not.
[box for 4 max lychee.zip](https://github.com/sn4k3/UVtools/files/11499665/box.for.4.max.lychee.zip)
![image](https://github.com/sn4k3/UVtools/assets/4185392/75d54d58-07db-4c8c-8b5d-d83f3fdacb71)

**Comment by bryanseger** [comment]
Welp, I'm getting a similar issue with Lychee and UVtools for the Mars 4 DLP as well. 
[cube lychee DLP.zip](https://github.com/sn4k3/UVtools/files/11499730/cube.lychee.DLP.zip)

Attached is the DLP file that UVtools has an error with opening.

So to summarize, Tango and UVtools play nicely for both Mars 4 machines, but Lychee and UVtools do not.

**Reply by sn4k3** [reply]
Lychee may use different checksum that cause that.
The checksum UVtools uses it's consistent with the GOO spec, so I guess lychee got it wrong. 
If printer still prints it probably don't care about checksum at all.

You should notify lychee about that.

**Reply by bryanseger** [reply]
Ok great, thanks for looking into it. 

I have notified them on their Discord, so hopefully they'll get that corrected.

In the meantime, do you see any potential workaround for getting UVtools to work with Lychee for the Mars 4? Or, am I stuck until Lychee fixes it on their end? (and to be clear, I can get Tango and UVtools to work, I just prefer Lychee over Tango)

**Reply by sn4k3** [reply]
There's no way to bypass the error other than fork UVtools, modify to your own and ignore the checksum validation.

**Comment by Cainam-git** [comment]
I had the same problem and contacted Lychee about it.

While I did not receive an answer yet, they just released version 5.2.1 which fixes this issue for me with the Mars 4 Max printer profile.

It's GOOD to finally be able to validate slices with UVTools! 😄
